service/src/test/java/com/ge/predix/controller/test/PolicyEvalWithGraphDbControllerIT.java

```diff
@@ -154,12 +170,82 @@ public void testPolicyEvaluation(final Zone zone, final PolicySet testPolicySet,
         assertThat(policyEvalResult.getEffect(), equalTo(expectedEffect));
     }
 
+    @Test(dataProvider = "policyEvalExceedingAttributeLimitDataProvider")
```

Comment:
this test is only run against one data set. Will be easier to read if there were no testng dataprovider.